web-dev/src/components/LoginPage/index.js

```diff
@@ -1,111 +1,88 @@
-                                            </div>
-                                        </div>
-                                        {/*Password*/}
-                                        <div className="login-user-frame">
-                                            <div className="position-relative">
-                                                <div className="login-email-box">
-                                                    <label htmlFor="login-password"
-                                                           className="login-user-label-title text-white-50">
-                                                        Password
-                                                    </label>
-                                                    <input type="password"
-                                                           ref={passwordRef}
-                                                           autoComplete="on"
-                                                           placeholder="Password"
-                                                           className="login-user-input text-white"
-                                                           id="login-password"
-                                                           required
-                                                    />
-                                                </div>
-                                            </div>
-                                        </div>
-                                        <button onClick={handleSignin}
-                                            className="btn login-button btn-submit btn-small"
-                                                type="submit">
-                                            Sign In
-                                        </button>
-                                        <div className="mt-3 mb-3">
-                                            <Link to="/registration" className="login-registration-link text-white-50">
-                                                New to Netflicks?
-                                                <span className="text-white"> Sign up now. </span>
-                                            </Link>
-                                        </div>
-                                        <div className="mt-3 mb-3">
-                                            <Link to="/homepage" className="login-registration-link text-white-50">
-                                                Or
-                                                <span className="text-white"> Guest </span> for now.
-                                            </Link>
-                                        </div>
-                                    </form>
-                                </div>
-                            </div>
+  const emailRef = useRef();
+  const passwordRef = useRef();
+  const {signIn} = useAuth()
+  const handleSignin = event => {
```

Comment:
I remove the navigate stuff here and use the one provided automatically by UnauthenticatedLock